package.json

```diff
@@ -31,5 +32,8 @@
   },
   "package-deps": [
     "linter"
-  ]
+  ],
+  "devDependencies": {
+    "openzeppelin-solidity": "^1.10.0"
```

Comment:
Please remove

Reply:
It's quite useful for testing purposes and I believe atom will not install dev dependencies. I've added open zeppelin contract from node_modules into sample.sol